src/components/orders/OrdersUserDetailsTable/index.tsx

```diff
@@ -150,6 +154,10 @@ const RowOrder: React.FC<RowProps> = ({ order, isPriceInverted }) => {
     setIsPriceInverted((previousValue) => !previousValue)
   }
 
+  const renderSpinnerIfDash = (textValue: string): JSX.Element | void => {
```

Comment:
The name is a bit unclear.

What about `renderSpinnerWhenNoValue` or something like that?